openpype/hosts/houdini/api/plugin.py

```diff
@@ -247,9 +247,22 @@ def remove_instances(self, instances):
         """
         for instance in instances:
             instance_node = hou.node(instance.data.get("instance_node"))
+            node = instance.data.get("instance_node")
+
             if instance_node:
                 instance_node.destroy()
 
+            # for the extra render node from the plugins
+            # such as vray and redshift
+            ipr_node = hou.node("{}{}".format(node,
+                                              "_IPR"))
+            if ipr_node:
+                ipr_node.destroy()
+            re_node = hou.node("{}{}".format(node,
+                                             "_render_element"))
+            if re_node:
+                re_node.destroy()
```

Comment:
I feel like this shouldn't be on the base `HoudiniCreator`. It's very specific to renderer creators (and I believe within those only a few of them do that.)

I'd say this remove logic should be added into the respective creators which need it. If there's a huge overlap for 'renderer' rops then maybe it's worth making a `HoudiniRenderCreator` base class if needed - but as far as I know there isn't that much overlap actually.